substrate/frame/revive/Cargo.toml

```diff
@@ -39,7 +39,7 @@ frame-benchmarking = { optional = true, workspace = true }
 frame-support = { workspace = true }
 frame-system = { workspace = true }
 pallet-revive-fixtures = { workspace = true, optional = true }
-pallet-revive-uapi = { workspace = true, features = ["scale"] }
+pallet-revive-uapi = { workspace = true, features = ["scale", "unstable-api"] }
```

Comment:
Should not be necessary. Only the fixtures make use of the host functions. The pallet itself just uses some types like ` ReturnFlags`.